packages/app/src/screens/RoutineEditScreen.tsx

```diff
@@ -1,18 +1,40 @@
-import { useNavigation } from '@react-navigation/native';
-import TaskItem from 'components/MyPlan/Routine/TaskItem';
-import React from 'react';
+import React, { useState } from 'react';
 import { Pressable, StyleSheet, View } from 'react-native';
 import { SvgXml } from 'react-native-svg';
 
+import TaskItem from 'components/MyPlan/Routine/TaskItem';
 import { colors } from 'design-token';
-import { icon_add, task } from 'icons';
-import { Divider, FixedBottomCTA, Input, Stack, Text } from 'ui';
+import { icon_add } from 'icons';
+import { Divider, FixedBottomCTA, Input, SlideUpModal, Stack, Text } from 'ui';
+import HistoryModal from 'components/MyPlan/Routine/HistoryModal';
+import AddTaskModal from 'components/MyPlan/Routine/AddTaskModal';
+import { Task } from 'types';
+import WheelPicker from 'ui/src/components/WheelPicker';
 
 const RoutineEditScreen = () => {
-  const taskData = [];
+  const [isHistoryModalVisible, setHistoryModalVisible] = useState(false);
+  const toggleHistoryModal = () => {
+    setHistoryModalVisible(!isHistoryModalVisible);
+  };
```

Comment:
```suggestion
  const toggleHistoryModal = (visible: boolean) => {
    // ...visible에 따른 기타 작업
    setHistoryModalVisible(visible);
  };
```


`toggleHistoryModal()`과 `toggleAddTaskModal()`의 용도를 잘 모르겠어요😢
버튼이 눌렸을 땐 이 함수들을 사용하지만, `Modal` 컴포넌트의 `setModalVisible`에는 `setHistoryModalVisible`을 전달하고 있네요.

모달이 제거될 때 입력했던 정보를 초기화 한다거나 하는 등의 추가적인 작업이 있을 경우에는 `Modal` 컴포넌트의 `setModalVisible`의 파라미터 `visiable`를 통해서 해야 할 것 같아요.
그럼 버튼 컴포넌트의 `onPress`에는 `setHistoryModalVisible(true)`를 실행하고, 
`Modal`의 `setModalVisiable`에 `toggleHistoryModal(visiable)` 함수를 실행해야 하는 게 맞지 않을까용?

Reply:
왜때문에 이렇게 했었는지 모르겠네요 ㅠㅠ !! 말씀하신 부분이 맞는것 같아요. 불필요하게 함수를 추가했네요!